kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ClusterService.java

```diff
@@ -45,7 +44,7 @@ public ClusterService() {
      */
     static String serviceName(VirtualKafkaCluster cluster) {
         Objects.requireNonNull(cluster);
-        return name(cluster);
+        return ResourcesUtil.name(cluster);
```

Comment:
One of the classes in their AbstractDependentResource class hierarchy gained a `name` method, which means unqualified calls to `ResourceUtils.name` now require qualification.